README.md

```diff
@@ -1,12 +1,26 @@
-[![AWS Kubernetes Integration Test](https://github.com/scalar-labs/scalar-terratest/workflows/Integration-test-with-terratest-for-AWS-Kubernetes/badge.svg?branch=master)](https://github.com/scalar-labs/scalar-terratest/actions)
+This repository contains a set of [documents](./docs/) and [configuration samples](./conf/) that help you create a ScalarDB or ScalarDL environment on managed Kubernetes services.
 
-[![Azure Kubernetes Integration Test](https://github.com/scalar-labs/scalar-terratest/workflows/Integration-test-with-terratest-for-Azure-Kubernetes/badge.svg?branch=master)](https://github.com/scalar-labs/scalar-terratest/actions)
+# Index
 
```

Comment:
Sorry, I must have not selected "Add review comment" for my change to this part.

Let's delete this heading (`# Index`) since we list the types of content below. (We can keep the headings I changed for `## Deployment guides` and subsequent headings as is).

```suggestion
```